package.json

```diff
@@ -8,7 +8,7 @@
     "build": "yarn build:rollup && yarn build:next",
     "build:next": "cd ./site && next build && next export",
     "build:rollup": "rollup --config ./config/rollup/rollup.config.js",
-    "clean": "rimraf ./node_modules ./packages/*/{dist,lib,node_modules} ./site/{.next,out}",
+    "clean": "bash -c 'rimraf ./packages/*/{dist,lib,node_modules} ./site/{.next,out}'",
```

Comment:
What about windows users? I guess this probably already didn't work for them unless they're in WSL.

Reply:
I had the same thought, but came to conclusion that either work on Windows without WSL.

Reply:
rimraf is a cross platform compatible rm -rf utility library, so this should have been working on windows unless there's something I'm missing.

Reply:
Rimraf isn’t the problem here, it’s the bash syntax to select which folders to remove. This already didn’t work on Windows command line.

Reply:
I'll appeal to a bit of authority here and say that I originally wrote this command on a windows box 😆  so I'm pretty confident the old version of it works. 

`rimraf` is built on top of a `glob` parser that has direct support for this notation https://github.com/isaacs/node-glob#glob-primer

Reply:
I'll appeal to a bit of authority here and say that I originally wrote this command on a windows box 😆 so I'm pretty confident the old version of it works.

rimraf is built on top of a glob parser that has direct support for this notation https://github.com/isaacs/node-glob#glob-primer

Reply:
Oh 🤔 I guess it makes sense glob would support the shell syntax for this as well. Carry on 😆

Reply:
Assuming this was done because Yarn threw an error:
You need to quote the glob pattern to avoid glob expansion

Reply:
Ah i didn't realize rimraf uses its own glob parser, makes sense. Thanks, will fix it up.